tests/unittest/_torch/modeling/test_modeling_gemma2.py

```
@@ -0,0 +1,135 @@
+import unittest
+from copy import deepcopy
+
+import torch
+from transformers import Gemma2Config
+from transformers import Gemma2ForCausalLM as HFGemma2ForCausalLM
+
+import tensorrt_llm
```

Flake8 (7.3.0)

[error] 'tensorrt_llm' imported but unused

(F401)